src/module/exports.rs

```diff
@@ -49,6 +51,10 @@ pub enum ExportItem {
 pub struct ModuleExports {
     /// The arena containing this module's exports.
     arena: TombstoneArena<Export>,
+    fn_id_to_export_id: HashMap<FunctionId, ExportId>,
+    tbl_id_to_export_id: HashMap<TableId, ExportId>,
+    mem_id_to_export_id: HashMap<MemoryId, ExportId>,
+    global_id_to_export_id: HashMap<GlobalId, ExportId>,
```

Comment:
Instead of using the default `HashMap` with the default hasher, we have a faster version for specifically working with ids:

https://github.com/rustwasm/walrus/blob/42d3b6116568798309797dad4ede9880835dbecc/src/map.rs#L18-L19